style.css

```diff
@@ -0,0 +1,228 @@
+body {
+    height: 100vh;
+    width: 100vw;
+    background-color: darkred;
+    margin: 0;
+}
+
+.header1 {
+    background-color: black;
+    height: 20vh;
+    width: 100vw;
+    /* position: absolute; */
+    /* top: 0; */
+
+}
+
+.headerImage {
+    position: relative;
+    left: 50%;
+    transform: translateX(-50%);
+    height: 84%;
+    width: auto;
+}
+
+main {
+    height: 80vh;
+    font-size: 0;
+}
+
+.playerBoxBorder{
+    height: 100%;
+    width: 25%;
+    display: inline-block;
+}
+
+.playerBox {
+    position: relative;
+    top: 7%;
+    left: 50%;
+    transform: translateX(-50%);
+}
+
+.playerBox1 {
+    width: 15vw;
+    background-color: darkslategray;
+    border-radius: 8px;
+    border: 1px solid red;
+    /* position: relative; */
+    height: 60vh;
+    text-align: center;
+    /* margin-left: 25px; */
+    /* top: 5vh; */
+    display: inline-block;
+    vertical-align: top;
+}
+
+.statsBox {
+    position: relative;
+    top: 13%;
+    height: 60%;
+    color: #331c14;
+    /* background-color: lime; */
+}
+
+#player1 {
+    width: 6em;
+    height: 2em;
+}
+
+.forfeitButton {
+    position: relative;
+    top: 13%;
+}
+
+.playerBox2 {
+    width: 15vw;
+    background-color: darkslategray;
+    border-radius: 8px;
+    border: 1px solid red;
+    /* position: relative; */
+    height: 60vh;
+    text-align: center;
+    /* left: 82vw; */
+    /* bottom: 60vw; */
+    display: inline-block;
+    vertical-align: top;
+
+}
+.pieceDisplay1 {
+
+    background-color: darkslategray;
+    /* height: 20vh; */
+    width: 10vw;
+    position: relative;
+    top: 4vh;
+    left: .5vw;
+
+}
+.pieceDisplay2 {
+
+    background-color: darkslategray;
+    height: 20vh;
+    width: 10vw;
+    position: relative;
+    left: .5vw;
+
+
+}
+
+#dpgamepiece {
+    position: relative;
+    top: 3.5vh;
+}
+
+img {
+    position: relative;
+    left: 20%;
+    width: 60%;
+}
+
+.gameArea {
+    /* display: inline-block; */
+}
+
+.gameBoard {
+    width: 50vmin;
+    height: 50vmin;
+    position: relative;
+    /* left: 25vw; */
+    margin: 0;
+    /* bottom: 5vw; */
+    display: inline-block;
+    vertical-align: top;
+    top: 14%;
+}
+.row0, .row1, .row2, .row3, .row4, .row5, .row6 {
+    /* position: relative; */
+    display: inline-block;
+    background-color: darkred;
+    height: 14.2857143%;
```

Comment:
that is... very exact... why?

And why not add a common class of row to everything that needed this?